Dockerfile

```diff
@@ -1,8 +1,35 @@
-FROM alpine:3.7
+FROM alpine:3.7 as base
+# Install go binary for make bin/gcsproxy
+COPY --from=golang:1.13-alpine /usr/local/go/ /usr/local/go/
```

Comment:
could we not just use `golang:1.13-alpine3` as *FROM* rather than doing this weird copy ?